### PR TITLE
SSU8: Strangling the monolith

### PR DESCRIPTION
**Pull request by Alex-Yates**
# About

This is a post about...

# Release

Is this post for a specific release? If so which one?

# Blog image idea

The design team will work on a design for the blog image. Add any ideas you have about the graphic image here.

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://style.octopus.com/writing-at-octopus-tldr).
- [ ] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [ ] The images I've included follow the [rules for working with images](https://style.octopus.com/images#rules-for-screenshots).

### CODE REVIEW

File: blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+---
+title: Safe schema updates - Strangling the monolith
+description: This post discusses patterns for migrating from a tightly coupled to a loosely coupled architecture
```
Comment (TeganAli) [inline blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md:3]:
```suggestion
description: This post discusses patterns for migrating from a tightly coupled to a loosely coupled architecture.
```

File: blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+!include <safe-schema-updates-posts>
+
+So far in this series we’ve talked at length about the benefits of embracing a more loosely coupled architecture, where sub-systems manage their own data. By hiding the database internals for each sub-system, and forcing all communication to go through some messaging bus or API gateway, we avoid the dependency nightmares discussed in database delivery hell. We also allow sub-systems to be managed and updated independently, reducing governance overhead, and simplifying both provisioning and delivery challenges.
+
+In the last post [we discussed patterns for safe, near-zero downtime releases](https://octopus.com/blog/safe-schema-updates-1-delivery-hell), with fast and reliable rollback options. You’re armed with the motivation and tooling, but we’ve not discussed any guiding strategy for planning the refactor from tight coupling to loose coupling.
+
+This post aims to provide a broad overview about how to manage the whole process. As the title suggests, we’ll discuss [the strangler pattern](MARTING FOWLER BLOG POST), but we’ll also touch on other patterns including:
```
Comment (TeganAli) [inline blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md:24]: @Alex-Yates do you have the link for this? Thanks :)

File: blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+metaImage: 
+bannerImage: 
+bannerImageAlt: Octopus strangling a database
+isFeatured: false
+tags:
+ - DevOps
+ - Database Deployments
+ - Deployment Patterns
+---
+This blog post is part 8 of my safe schema updates series. We'll add links to the other posts in this series as they become available:
```
Comment (TeganAli) [inline blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md:16]: @Alex-Yates just double-checking this is the last post of the series, before I delete the line about adding links to other posts? Thanks :)

File: blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md (hunk 2 above)
Comment (Alex-Yates) [reply blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md:24]: Oops, my bad! https://martinfowler.com/bliki/StranglerFigApplication.html

File: blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md (hunk 3 above)
Comment (Alex-Yates) [reply blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md:16]: Yep this is it. All done! :-)

File: blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md (hunk 2 above)
Comment (TeganAli) [reply blog/2021-12/safe-schema-updates-8-strangling-the-monolith/index.md:24]: Thank you :)

**Pull request merged**